### DISCUSSION TITLE
Why XFS as filesystem for backing storage?

### DISCUSSION THREAD

**Discussion by KrzysztofHajdamowicz** [marcone/teslausb#779] [General]
Why XFS was chosen as filesystem for backing storage?
Looking at https://github.com/marcone/teslausb/blob/main-dev/run/make_snapshot.sh#L202 backing file is copied as a snapshot.

**Comment by marcone** [comment]
> backing file is copied as a snapshot.

which is why it uses xfs, since that supports snapshots. I initially used btrfs, but it had trouble with hard shutdown that caused it to "lose" disk space over time, and xfs proved to be more robust.
